doc/blackbird_python/example.rst

```diff
@@ -132,3 +132,62 @@ using :func:`~.dump`:
     with open('new_file.xbb', 'w') as f:
         blackbird.dump(bb, f)
 
+
+Templates
+---------
+
+Templated Blackbird scripts, with templated gate parameters, are saved as ``.xbt`` files.
```

Comment:
Do we need to have another suffix? Can we not just allow regular `.xbb` files to be templated?

Reply:
Sure.
```suggestion
Templated Blackbird scripts, with templated gate parameters, are saved as ``.xbb`` files.
```